bitmark-references/bitmark-cli/bitmark-cli.md

```diff
@@ -0,0 +1,729 @@
+# bitmark-cli Command Reference
+
+## Basic Command Structure
```

Comment:
I am personally hoping that we could have full commands for some common functions that I just need to copy & paste to execute at the beginning. Example:
- setup the CLI
- Add an identity ( account)
- Compute fingerprint for an asset
- Issue a bitmark 
- Verify the bitmark status
- Transfer a bitmark (might also pay for that transaction).

Reply:
And in the command details section, we could explain each option and add advance uses.

Reply:
perhaps a separate example section

Reply:
stared in quick-setup.md